doc/binder/requirements.txt

```diff
@@ -0,0 +1,4 @@
+-r ../../requirements-build-docs.txt
+nilearn==0.6.0  # Need to update on each PyPi release
```

Comment:
Why `==0.6.0`? Shouldn't any stable version of Nilearn `>=0.6.0` work?

Reply:
My original idea was to pin to the latest PyPi release. There's no strict reason we need to do that, though, it's mostly a choice. TBH, most other projects I've seen implementing something like this just build from master. WDYT ?

Reply:
I thought you were getting errors when using master?

Reply:
I think so long as we keep the specs `>=0.6.0` it wouldn't matter. But pinning it to `0.6.0` will add release fault line.

Reply:
> I thought you were getting errors when using master?

No, not that I recall at least ! I just was thinking that if we build from master then the binder examples will update with github rather than the docs. Not sure if that's really a problem, but the alternative (pinning to the last pypi release, as I have here) will require updating the release guide -- if one exists ?!

Reply:
If we make it `>=0.6.0` it will always onstall the latest stable version, which is what the docs corresponds to and we won't have to update this manually at every release.

Reply:
Ah ok, I understand. Could we capture this behavior a little more transparently by passing the `-U` flag ?

Reply:
I believe `>=version number` is plenty transparent.
`-U` will explicitly upgrade Nilearn in whichever environment we install it, to the latest version, which may not be the user's intention. `>=` will only upgrade it if it is less that `0.6.0`.

PS: I will soon release a `0.6.1` version to backport the OSF redirect error fix.

Reply:
`0.6.1` released. 
Think we should set that as the minimum?